setup.py

```diff
@@ -36,11 +36,6 @@
             "teams = lms.djangoapps.teams.plugins:TeamsTab",
             "textbooks = lms.djangoapps.courseware.tabs:TextbookTabs",
             "wiki = lms.djangoapps.course_wiki.tab:WikiTab",
-
-            # ORA 1 tabs (deprecated)
-            "peer_grading = lms.djangoapps.open_ended_grading.views:PeerGradingTab",
-            "staff_grading = lms.djangoapps.open_ended_grading.views:StaffGradingTab",
-            "open_ended = lms.djangoapps.open_ended_grading.views:OpenEndedGradingTab",
```

Comment:
If we want to leave the code around and just not use it, I would recommend changing `is_enabled` for the tabs in `open_ended_grading/views.py` to return False every time. This will cause the code to fail to find the tabs, leaving an error that will make the logs noisy.